src/pages/Onboard/Onboard-Page.vue

```diff
@@ -169,7 +176,8 @@ export default {
 
     <v-container class="position-absolute onboard-content pa-0" fluid>
       <transition name="fade" mode="out-in">
-        <router-view class="router-view" />
+        <acceptPage v-if="$route.name == 'accept'" />
```

Comment:
Wouldn't it make more sense to add `accept` as a child route of `welcome`? Then this portion will be taken care of automatically and we won't have 2 routes pointing to the same component

Reply:
Yes we can.  I was going to do that as part of a separate PR though as the invitation email currently sends to _cloud.prefect.io/accept_ so it would involve changing the routing either as part of a navguard or watch on app or as a PR to Cloud.